mp/src/game/server/momentum/mom_powerup_damageboost.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "cbase.h"
```

Comment:
It's typically bad practice to include this in a header since it's the PCH and contains some heavy hitters like windows includes and stuff. You're probably looking for
```suggestion
#include "baseanimating.h"
```
Or something?

Reply:
done.